### PR TITLE
Add nexus config and artifacts

### PR DESCRIPTION
**Pull request by nambrot**

### CODE REVIEW

File: deployments/ism/nexus/config.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# yaml-language-server: $schema=./core-deployments.schema.json
+---
+description: ISM config for Nexus
+timestamp: 2023-09-23T16:00:00.000Z
```
Comment (jmrossy) [inline deployments/ism/nexus/config.yaml:4]: Is this timestamp right?

File: deployments/warp_routes/nexus_arbitrum/config.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+a# yaml-language-server: $schema=./core-deployments.schema.json
```
Comment (jmrossy) [inline deployments/warp_routes/nexus_arbitrum/config.yaml:1]: typo leading `a` here

**Comment by jmrossy** [comment]
No objections to the schema here though I'm confused why you included just the warp + ism configs/artifacts and not the core artifacts for these chains

**Comment by nambrot** [comment]
The main goal was to have a reference for the warp routes because that was blocking folks. I suspect core artifacts can come later

File: deployments/ism/nexus/config.yaml (hunk 1 above)
Comment (nambrot) [reply deployments/ism/nexus/config.yaml:4]: No, but I believe currently unused? I just had it copy-pasted

**Pull request merged**